From affa020559bca31d6531e19cb1f009c22705a73d Mon Sep 17 00:00:00 2001
From: Hans de Goede <hdegoede@redhat.com>
Date: Thu, 31 Mar 2016 14:38:14 +0200
Subject: [PATCH 2/2] sunxi: Enable LDO3 and LDO4 at 2.8V on OLinuxIno Lime
 boards

LDO3 and LDO4 are used to power port E resp. port G, which are exposed
on gpio headers, so enable them at 2.8V as specified in the schematic.

Signed-off-by: Hans de Goede <hdegoede@redhat.com>
---
 configs/A10-OLinuXino-Lime_defconfig  | 2 ++
 configs/A20-OLinuXino-Lime2_defconfig | 2 ++
 configs/A20-OLinuXino-Lime_defconfig  | 2 ++
 3 files changed, 6 insertions(+)

Index: u-boot/configs/A10-OLinuXino-Lime_defconfig
===================================================================
--- u-boot.orig/configs/A10-OLinuXino-Lime_defconfig
+++ u-boot/configs/A10-OLinuXino-Lime_defconfig
@@ -13,4 +13,6 @@ CONFIG_SYS_EXTRA_OPTIONS="SUNXI_EMAC,AHC
 # CONFIG_CMD_FLASH is not set
 # CONFIG_CMD_FPGA is not set
 CONFIG_CMD_GPIO=y
+CONFIG_AXP_ALDO3_VOLT=2800
+CONFIG_AXP_ALDO4_VOLT=2800
 CONFIG_USB_EHCI_HCD=y
Index: u-boot/configs/A20-OLinuXino-Lime2_defconfig
===================================================================
--- u-boot.orig/configs/A20-OLinuXino-Lime2_defconfig
+++ u-boot/configs/A20-OLinuXino-Lime2_defconfig
@@ -14,4 +14,6 @@ CONFIG_SYS_EXTRA_OPTIONS="SUNXI_GMAC,RGM
 # CONFIG_CMD_FPGA is not set
 CONFIG_CMD_GPIO=y
 CONFIG_ETH_DESIGNWARE=y
+CONFIG_AXP_ALDO3_VOLT=2800
+CONFIG_AXP_ALDO4_VOLT=2800
 CONFIG_USB_EHCI_HCD=y
Index: u-boot/configs/A20-OLinuXino-Lime_defconfig
===================================================================
--- u-boot.orig/configs/A20-OLinuXino-Lime_defconfig
+++ u-boot/configs/A20-OLinuXino-Lime_defconfig
@@ -12,4 +12,6 @@ CONFIG_SYS_EXTRA_OPTIONS="SUNXI_GMAC,AHC
 # CONFIG_CMD_FPGA is not set
 CONFIG_CMD_GPIO=y
 CONFIG_ETH_DESIGNWARE=y
+CONFIG_AXP_ALDO3_VOLT=2800
+CONFIG_AXP_ALDO4_VOLT=2800
 CONFIG_USB_EHCI_HCD=y
